recipe/meta.yaml

```diff
@@ -78,6 +76,7 @@ test:
         - sites.sbx
         - sites.shp
         - sites.shx
+        - cropped.cub
```

Comment:
That's fine, thanks for debugging this! All planetary python it's will be thankful if we can haz a stable GDAL package!